packages/manager/modules/sign-up/src/components/siret/siret.component.js

```diff
@@ -15,5 +15,6 @@ export default {
     mode: '<',
     fieldToFocus: '<',
     isIndianSubsidiary: '<?',
+    legalFormList: '<',
```

Comment:
Beware this component is not only used in the signup app, so by making this change you'll probably break the behavior in the other places it is used (at least new-account for account modification)